driver/src/main/java/eu/cloudnetservice/driver/service/ServiceEnvironmentType.java

```diff
@@ -104,6 +104,14 @@ public final class ServiceEnvironmentType implements DefaultedDocPropertyHolder,
     .name("MINESTOM")
     .properties(Document.newJsonDocument().writeProperty(JAVA_SERVER, true).writeProperty(PLUGIN_DIR, "extensions"))
     .build();
+  /**
+   * The Limbo service environment type (Java Edition dummy server).
+   */
+  public static final ServiceEnvironmentType LIMBO = ServiceEnvironmentType.builder()
```

Comment:
This environment is very specific to limbo loohp. while the name is very general. I'm wondering if this should be named more specifically to allow for other limbos to be added in the future as well (...maybe)

Reply:
LIMBO_LOOHP ? I'm not very creative with names.

Reply:
Sounds good, keep in mind to change it in the versions.file too